docs/tutorials/tutorial_3k_tcr.md

```diff
@@ -560,9 +563,6 @@ ir.pl.clonotype_imbalance(
     case_label="Tumor",
     additional_hue="diagnosis",
     plot_type="volcano",
+    fig_kws={"dpi": 120},
 )
 ```
```

Comment:
It would be really nice if you could pick the top "differentially enriched" clonotypes between cell-type annotations and show them next to the cluster labels on the UMAP plot. I think this would help to better understand the concept of differentially enriched. 
Does that make sense? Or am I missing something?

Reply:
I've started to do this. Sounds good, but will see after we've tried.